src/lib/rest/StringFilter.cpp

```diff
@@ -1255,7 +1328,20 @@ bool StringFilter::mongoFilterPopulate(std::string* errorStringP)
       left = itemP->attributeName;
     }
 
-    if (left == DATE_CREATED)
+ 
+    //
+    // Also, if the left hand side contains a compound path, then we are
+    // dealing with matching of a compound item.
+    //
+    if (itemP->compoundPath.size() != 0)
+    {
+      if (itemP->type == SftQ)
+      {
+        left = std::string(ENT_ATTRS) + "." + itemP->attributeName + ".value." + itemP->compoundPath;
```

Comment:
Ah, of course, there is a define in there somewhere for "value".
No probs, I'll look it up.